index.spec.js

```diff
@@ -0,0 +1,5 @@
+import directive from './index';
+
+it('getDirectiveDeclaration should be defined', () => {
+  expect(directive.getDirectiveDeclaration()).toMatchSnapshot();
```

Comment:
Maybe we can add an example of how this is helpful in user-land?

Reply:
As a method name suggest. If the direction has declaration the user doesn't have to declare it by itself.